po/Makevars

```diff
@@ -57,7 +57,7 @@ USE_MSGCTXT = no
 # Useful options are in particular:
 #   --previous            to keep previous msgids of translated messages,
 #   --quiet               to reduce the verbosity.
-MSGMERGE_OPTIONS = --add-location=never
+MSGMERGE_OPTIONS = --no-location
```

Comment:
It looks like the ability to take a parameter for add-location was only added in mid-2014, which would explain why it isn't in 14.04: https://lists.gnu.org/archive/html/bug-gettext/2014-03/msg00021.html

What is the difference between --add-location=never and --no-location? My reading of the man page was that they were identical...